### ISSUE TITLE
I can't find doc/rdoc/markup_reference.rb in the https://ruby.github.io/rdoc/

### ISSUE THREAD

**Issue by sonysantos** [ruby/rdoc#905]
I think the best RDoc markup reference is in the file [doc/rdoc/markup_reference.rb](https://github.com/ruby/rdoc/blob/master/doc/rdoc/markup_reference.rb).

But it's hidden from the official documentation in https://ruby.github.io/rdoc/.

(Maybe it's because it's not in the `lib` folder? But, if that is the case, how do I config RDoc to include the `doc` folder into the output doc?)

I think it's a very important reference to be hidden from the documentation.

**Comment by BurdetteLamar** [comment]
First of all, thanks for "best RDoc reference"!

The file started out in lib/, but a reviewer rightly pointed out that b/c it's not real library code, it should not be put where it may be required.

So I moved it to doc/, not realizing that from there it wouldn't be copied into ruby/ruby.

Very recently (< 48 hours ago), I've moved it to doc/rdoc/ and modified the copying mechanism to get it into ruby/ruby, where it will become righteous documentation.  I'm not sure how long it takes to propagate into the doc, but if it doesn't turn up soon, I'll know there's a problem.

**Comment by BurdetteLamar** [comment]
https://github.com/ruby/ruby/blob/master/doc/rdoc/markup_reference.rb appeared in ruby/ruby just 18 hours ago, so there may yet be hope.

**Comment by sonysantos** [comment]
I can find the updated version at https://docs.ruby-lang.org/en/master/RDoc/MarkupReference.html.

But not in https://ruby.github.io/rdoc/ yet.

The first link is enough for what I need, thank you. However, I still expect that will be in the last reference soon, since the first one is not so obvious/trivial to reach directly from the RDoc repository.

**Comment by BurdetteLamar** [comment]
@sonysantos, may not have seen a year ago that this first cut covers only what was covered in the reference in the old [Rdoc reference](https://docs.ruby-lang.org/en/master/RDoc/Markup.html#class-RDoc::Markup-label-RDoc+Markup+Reference).  There's much much more that can be done.

**Comment by nobu** [comment]
I didn't know that domain.
But it says "rdoc 6.0.4 Documentaion", which is quite old.
Who created and maintains it?

**Comment by hsbt** [comment]
`gh-pages` is abandoned now. I'll migrate it with GHA.

**Comment by sonysantos** [comment]
Only for clarification, I've reached that page from the github repository main page.

![image](https://user-images.githubusercontent.com/22578228/181516301-f9ef8cc3-7bdb-4a8d-8c88-d41bf67bc362.png)

**Comment by nevans** [comment]
I think that maybe `gh-pages` isn't the default approach anymore?  And it would be very good to add an action triggered by changes to `master`.  But we can still use the classic `gh-pages` approach.  I think we need to add a `.nojekyll` file.  And I personally don't like the `rm_rf "*"` that's in the current task, so I think the build should use `/docs` instead of the root.  But other than that, it should work.

I updated the rake task with `.nojekyll` and some other safeguards here: #935.  And I've verified that it works just fine here: https://nevans.github.io/rdoc/
